Brief-over-of-Heads-and-firmware-security/Heads-threat-model.md

```diff
@@ -0,0 +1,471 @@
+ the system.
+
+* [LightEater malware seek GPG keys in Tails, Kallenberg and Kovah, 2015](https://www.theregister.com/2015/03/19/cansecwest_talk_bioses_hack/)
+* [Thunderstrike, Hudson 2014](https://trmm.net/Thunderstrike)
+
+For these reasons, Tails is not sufficient for many users who want a laptop that
+ they can travel with and want to have some assurances that most adversaries
+ won't be able to modify the hardware underneath them.
+
+Complicating this goal is that modern x86 hardware is full of modifiable state
+ [State considered harmful, Rutkowska 2015](https://blog.invisiblethings.org/2015/12/23/state_harmful.html)
+ and it is full of dusty corners that can hide malware or unauthorized code.
+ Additionally there is unverifable code running in the Intel Management Engine,
+ which has access to memory, to the network and various other peripherals. As a
+ result we must trust certain entities more than others and this does affect our
+ threat model.
+
+This document discusses some of the threats that make building slightly more
+ secure mobile systems very difficult. There is a separate Heads FAQ as well as
+ a guide on installing Heads on the Thinkpad x230 or Chell Chromebook, which
+ covers the practical issues of hardening a laptop against some of the threats
+ described here.
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+Who do we trust
+===
+
+![Flashing Chips]({{ site.url }}/images/Flash_chips.jpg)
+
+As we consider building secure hardware, it is very important to keep in mind
+ that there are some parties that we are forced to trust to some degree. There
+ are countermeasures (discussed later) that we can take to reduce the amount of
+ trust that we place in some of these parties, as well as ways to verify that
+ they are well behaved. Unfortunately the root of trust in the CPU manufacturer
+ is hard to get around.
+
+* Intel (or other CPU manufacturer)
+  * Management Engine
+  * Microcode updates
+  * Both are opaque and unknown outside of Intel
+* TPM manufacturer
+  * Infineon/STM/etc
+  * Might have a bad RNG, undocumented commands to leak keys, etc
+  * Might have government manadated backdoor (country dependent)
+  * LPC or i2c bus can be snooped
+* Laptop manufacturer
+  * Backdoors or malware in the BIOS [Computrace](https://www.kaspersky.com/about/press-releases/2014_good-software-can-go-bad)
+  [Lenovo Malware](https://thehackernews.com/2015/08/lenovo-rootkit-malware.html)
+  * Did they install extra devices on the LPC bus?
+  * Is there a key logger built into the keyboard?
+  * Funtennas for data leakage through motherboard traces
+  * [Funtenna uses software to make embedded devices broadcast data on radio frequencies, Ang Cui 2015)](http://www.slate.com/blogs/future_tense/2015/08/05/_funtenna_uses_software_to_make_embedded_devices_broadcast_data_on_radio.html)
```

Comment:
Funtenna ref is here for above

Reply:
Specifically https://github.com/osresearch/heads-wiki/pull/49/files/1899083cd24a471d83b67df360b4a174165d8bd3#diff-a9fcdfdf121c9e5fe20d9a12db884bd5R74